regions_www/m/server.R

```diff
@@ -358,6 +386,19 @@ shinyServer(function(input, output, session) {
     shinyjs::hideElement(id = "loading")
   }, priority = 3)
 
+  observe({
+
+    if(!is.null(region$state_region) && !is.na(region$state_region)){
+
+      leafletProxy("map") %>% setView(.,
+                                      lng = region$state_lng,
+                                      lat = region$state_lat,
+                                      zoom = region$state_mzoom
+      )
```

Comment:
Can this be done in `onRestore`?

Reply:
No, it can't be done there. `onRestore` runs before any observe block is executed - since leaflet variable hasn't been initialized, this can't work. 
I now have moved this block to the `onRestored` function which is run after all observe blocks are executed.